src/components/ReviewsBox/index.js

```diff
@@ -1,9 +1,39 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
 import ReviewBox from "../ReviewBox";
 
 const ReviewsBox = props => {
-  const { reviews } = props;
+  const { focusedPlaceId, getPlaceReview } = props;
+
+  const [reviews, setRreviews] = useState([]);
+  const [page, setPage] = useState(0);
+  const [hasNextPage, setHasNextPage] = useState(false);
+
+  const PAGE_PER_COUNT = 10;
+
+  useEffect(() => {
+    if (focusedPlaceId) {
+      getPlaceReview(focusedPlaceId, page).then(response => {
+        if (!(response instanceof Error)) {
+          const remainReviews = response.totalReview - PAGE_PER_COUNT * page;
+          if (remainReviews < 0) setHasNextPage(false);
+          else setHasNextPage(true);
+
+          setRreviews([...response.reviews]);
+        }
+      });
```

Comment:
catch를 사용하지 않은 이유가 있을까용?? 
```suggestion
      getPlaceReview(focusedPlaceId, page).then(({totalReview, reviews}) => {
          const isRemainNextPage = !!(totalReview - PAGE_PER_COUNT * page);
          setHasNextPage(isRemainNextPage);
          setRreviews([...reviews]); //오타발견 ㅎㅎ 
        }
      }).catch(err => { /*에러에 대한 처리*/})
```

Reply:
api 를 추상화 시켜둔 함수에서 사용하는 제가 만든 fetchHelper가 에러를 catch해서 바로 response로 error 객체를 넘겨주기 때문에 이렇게 해봤습니다. 물론 좀 이상하게 흘러가긴 했는데 어떤 방식으로 해결하는게 좋을지 감이 안오네요 😢